src/js/components/Box/stories/ResponsiveContainer.js

```diff
@@ -0,0 +1,118 @@
+import React, { useContext, useEffect } from 'react';
+
+import {
+  Box,
+  Button,
+  Heading,
+  ResponsiveContext,
+  Text,
+  ThemeContext,
+} from 'grommet';
+import { FormNext, FormPrevious } from 'grommet-icons';
+
+// Some content to show the effects of responsive layout.
+// The heading will change size at the small breakpoint
+// and the child Text items will switch from row to column
+// layout at the small breakpoint.
+const Content = ({ title, children }) => {
+  const size = useContext(ResponsiveContext);
+  const theme = useContext(ThemeContext);
+  const smallBreakpoint = theme.global.breakpoints.small.value;
+  return (
+    <Box pad="medium">
+      <Heading>{title}</Heading>
+      <Box direction="row-responsive" gap={{ column: 'medium', row: 'xsmall' }}>
+        <Text>I am row-responsive</Text>
+        <Text>Small breakpoint: {smallBreakpoint}</Text>
+        {children}
+        <Text>
+          Current size:&nbsp;<Text weight="bold">{size}</Text>{' '}
+        </Text>
+      </Box>
+    </Box>
+  );
+};
+
+const App = ({ title, responsive = true }) => {
+  const widthRef = React.useRef(undefined);
+  const containerRef = React.useRef(undefined);
+
+  // track the width of the container just so we can display it as it changes
+  useEffect(() => {
+    let resizeObserver;
+    const element = containerRef.current;
+    if (typeof window !== 'undefined' && window.ResizeObserver) {
+      resizeObserver = new window.ResizeObserver((entries) => {
+        const entry = entries[0].borderBoxSize[0];
+        const width = entry?.inlineSize;
+        if (widthRef.current) {
+          widthRef.current.innerText = `Container width: ${width.toFixed()}px`;
+        }
+      });
+      if (element) {
+        resizeObserver.observe(containerRef.current);
+      }
+    } else {
+      // fallback for server side rendering
+      const width = containerRef.current.getBoundingClientRect();
```

Comment:
```suggestion
      const width = containerRef.current.getBoundingClientRect().width;
```

would they just want the width of the getBoundingClientRect()??

Reply:
Good point. Thanks. This is code a normal human can't hit but I was able to test the fix via the debugger.